mu-plugins/blocks/query-filter/src/view.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * WordPress dependencies
+ */
+import { store as wpStore } from '@wordpress/interactivity';
+
+// See https://github.com/WordPress/gutenberg/blob/37f52ae884a40f7cb77ac2484648b4e4ad973b59/packages/block-library/src/navigation/view-interactivity.js
+const focusableSelectors = [
+	'a[href]',
+	'input:not([disabled]):not([type="hidden"]):not([aria-hidden])',
+	'select:not([disabled]):not([aria-hidden])',
+	'textarea:not([disabled]):not([aria-hidden])',
+	'button:not([disabled]):not([aria-hidden])',
+	'[contenteditable]',
+	'[tabindex]:not([tabindex^="-"])',
+];
+
+function closeDropdown( store ) {
+	const { context } = store;
+	context.wporg.queryFilter.isOpen = false;
+	context.wporg.queryFilter.form?.reset();
+
+	const count = context.wporg.queryFilter.form?.querySelectorAll( 'input[type="checkbox"]:checked' ).length;
+	updateToggleLabel( store, count );
+}
+
+function updateToggleLabel( store, count ) {
+	const { context } = store;
+	const toggle = context.wporg.queryFilter.toggleButton;
+	if ( ! toggle ) {
+		return;
+	}
+	toggle.querySelector( 'span' ).innerText = count;
+	if ( count ) {
+		toggle.classList.add( 'has-filter-applied' );
```

Comment:
Doesn't look like `.has-filter-applied` is used for anything, maybe not required?

Reply:
👍🏻